cypress/integration/tests/administrator/credentials/proxy_crud.test.ts

```diff
@@ -20,18 +28,24 @@ describe("Validation of proxy credentials", () => {
         preservecookies();
     });
 
+    it("Creating proxy credentials and canceling without saving", () => {
+        proxyCreds.create(toBeCanceled);
+    });
+
     it("Creating proxy credentials", () => {
         proxyCreds.create();
     });
 
+    it.skip("Editing proxy credentials and cancelling without saving", () => {
```

Comment:
Why are we skipping it ?

Reply:
Because now this test is failing and I'm trying to figure out how to fix it